packages/perseus/src/validation.types.ts

```diff
@@ -236,58 +238,105 @@ export type PerseusTableRubric = {
 
 export type PerseusTableUserInput = ReadonlyArray<ReadonlyArray<string>>;
 
-export type Rubric =
-    | PerseusCategorizerScoringData
-    | PerseusCSProgramRubric
-    | PerseusDropdownRubric
-    | PerseusExpressionRubric
-    | PerseusGroupRubric
-    | PerseusGradedGroupRubric
-    | PerseusGradedGroupSetRubric
-    | PerseusGrapherRubric
-    | PerseusIFrameRubric
-    | PerseusInputNumberRubric
-    | PerseusInteractiveGraphRubric
-    | PerseusLabelImageRubric
-    | PerseusMatcherRubric
-    | PerseusMatrixRubric
-    | PerseusNumberLineScoringData
-    | PerseusNumericInputRubric
-    | PerseusOrdererRubric
-    | PerseusPlotterScoringData
-    | PerseusRadioRubric
-    | PerseusSorterRubric
-    | PerseusTableRubric;
-
-export type UserInput =
-    | PerseusCategorizerUserInput
-    | PerseusCSProgramUserInput
-    | PerseusDropdownUserInput
-    | PerseusExpressionUserInput
-    | PerseusGrapherUserInput
-    | PerseusGroupUserInput
-    | PerseusIFrameUserInput
-    | PerseusInputNumberUserInput
-    | PerseusInteractiveGraphUserInput
-    | PerseusLabelImageUserInput
-    | PerseusMatcherUserInput
-    | PerseusMatrixUserInput
-    | PerseusNumberLineUserInput
-    | PerseusNumericInputUserInput
-    | PerseusOrdererUserInput
-    | PerseusPlotterUserInput
-    | PerseusRadioUserInput
-    | PerseusSorterUserInput
-    | PerseusTableUserInput;
-
-export type UserInputMap = {[widgetId: string]: UserInput};
+export interface RubricRegistry {
```

Comment:
Should this be ScoringDataRegistry? Doesn't have the same ring, but we're trying to move away from the `rubric` vocab, right? Also, should the Rubric types below be changed to ScoringData types or will we just handle that after landing my renaming PR when we get back? Might make sense to plan the order in which we land PRs, though it might make sense just to end the week with a fully merged feature branch with all our changes combined.

Reply:
I plan to land this PR after your scoring name changes... so I'll be sure to update the instances of Rubric in this PR before landing.